lib/crumple.rb

```diff
@@ -10,10 +10,19 @@ def initialize(target_file)
       @dump_dir = get_dump_dir
     end
 
+    def change_dump_dir(new_dump_dir)
```

Comment:
Unused method argument - `new_dump_dir`. If it's necessary, use `_` or `_new_dump_dir` as an argument name to indicate that it won't be used. You can also write as `change_dump_dir(*)` if you want the method to accept any arguments but don't care about them.